packages/xrpl/test/models/MPTokenAuthorize.test.ts

```
@@ -1,6 +1,11 @@
-import { assert } from 'chai'
+import { MPTokenAuthorizeFlags } from '../../src'
+import { validateMPTokenAuthorize } from '../../src/models/transactions/mptokenAuthorize'
```

ESLint

[error] Missing file extension for "../../src/models/transactions/mptokenAuthorize"

(import/extensions)